aerleon/utils/config.py

```diff
@@ -19,55 +21,49 @@
     'exp_info': 2,
 }
 
+default_file = './aerleon.yml'
```

Comment:
https://peps.python.org/pep-0008/#constants
Surprised none of our stuff caught that this is a constant?

Reply:
I've fixed `config.DEFAULT_FILE` but let `config.defaults` remain incorrect for the moment. I'll put that in the follow up PR.